src/vivarium_census_prl_synth_pop/components/observers.py

```diff
@@ -716,6 +727,8 @@ def get_observation(self, event: Event) -> pd.DataFrame:
             df_w2[col] = df_w2["employer_id"].map(business_details[col])
 
         df_w2 = df_w2.set_index(["simulant_id"])
+
+        df_w2["is_w2"] = True
```

Comment:
No need to do anything about this now, but in future when you create a placeholder like this that will be modified in a subsequent PR, it's helpful to add a comment mentioning that this is the case so a reviewer doesn't try to understand why we're always setting this to True